### ISSUE TITLE
Cannot Record Streams Due to Missing "Write" Permission

### ISSUE THREAD

**Issue by SevenFactors** [segler-alex/RadioDroid#1190]
**Specs:**
This is on Pixel phone running GrapheneOS (latest) which is based on Android14

**Issue:**
I can't record.

When pressing the Record button I'm presented with a message at the bottom that reads:
**"Need write permission for recording"**

When going to App Info/Permissions. The only app permission that RadioDroid has are: Network, Notifications and Sensors.

RadioDroid is missing Android 14 support? Please add the missing permission necessary to allow stream recording.

Thanks

**Comment by SnoopAir** [comment]
I can confirm the same issue. I'm running on Android 12 still, but also can't record. The app is only saying "Need write permission for recording". But it never requests these (no pop up, nothing), nor can be added in the system permissions for the app.
So to me it seems it is not a Android 14 problem but a problem in general.

Thanks for looking into the issue.

**Comment by manzerbredes** [comment]
Same issue on Android 14 with Galaxy S22. Write permission is neither visible in app authorizations.

**Comment by Wikinaut** [comment]
Same here!

This is a severe regression.

**Comment by pascal-giese** [comment]
same issue running graphene os on pixel 7a

**Comment by s1dh** [comment]
same issue on Android 14

**Comment by ours1** [comment]
Hi ! Same same .... Asks for write permission but cannot find any way to set them up

**Comment by DLRY54** [comment]
Hello, same issue. Have 2 identical phones running on Android V13. Radiodroid V0.86. One can record streams the other one is missing record permission. Where to add or change the permission?
Thanks for this fantastic app

**Comment by mikael35** [comment]
> Hello, same issue. Have 2 identical phones running on Android V13. Radiodroid V0.86. One can record streams the other one is missing record permission. Where to add or change the permission? Thanks for this fantastic app

By reading https://github.com/segler-alex/RadioDroid/issues/1193 you maybe have upgrading them differently.

**Comment by aarek-eng** [comment]
Same problem here, Fairphone 4, Fairphone OS, based on Android 13 apparently.

**Comment by pascal-giese** [comment]
> > Hello, same issue. Have 2 identical phones running on Android V13. Radiodroid V0.86. One can record streams the other one is missing record permission. Where to add or change the permission? Thanks for this fantastic app
> 
> By reading https://github.com/segler-alex/RadioDroid/issues/1193I think you maybe have upgraded them differently.

This works! (after you delete the "I" after "1193" in the link. The solution: Install version 0.84, grant the permission and then update if you want to. Recording is working again.

**Comment by aarek-eng** [comment]
> > > Hell 
> This works! (after you delete the "I" after "1193" in the link. The solution: Install version 0.84, grant the permission and then update if you want to. Recording is working again.

That's less a 'solution' than a 'temporary workaround'. But I'll take it!

**Comment by pascal-giese** [comment]
true. but given that it keeps working after updating and app development seems dead; it will be a sad day sometime

**Comment by bobemoe** [comment]
The workaround helped me too. Here is the official download page. https://github.com/segler-alex/RadioDroid/releases/tag/0.84

I also found 0.84 is still in F-droid previous versions too.
